relayer/chain/chain.go

```diff
@@ -14,8 +14,15 @@ type Message struct {
 	Payload interface{}
 }
 
+type Header struct {
```

Comment:
We may want these types to be Ethereum specific, as for the other direction we're not really relaying headers. Can leave it as is for now and refactor once we get to implementing the other direction though.